### ISSUE TITLE
no credits give you a cryptic message

### ISSUE THREAD

**Issue by joamatab** [flexcompute/tidy3d#769]
```
HTTPError: 409 Client Error:  for url: https://tidy3d-api.simulation.cloud/tidy3d/tasks/fdve-e8475658-c228-4c9a-a3a5-4c379071e921v1/submit
```

![image](https://user-images.githubusercontent.com/4514346/226390420-d673397a-e77c-4a05-8ec4-498fa3977ed8.png)

**Comment by momchil-flex** [comment]
@An-Li-magicloud could you have a look at the way errors are handled by the python web api?
Or maybe this has been fixed in the webapi refactor in the upcoming 1.10?

**Comment by tylerflex** [comment]
Fixed by #817
